src/fastga_he/models/rta_variables/rta_aero_approximation.py

```diff
@@ -0,0 +1,275 @@
+            np.pi
+            * inputs["data:geometry:horizontal_tail:aspect_ratio"]
+            * inputs["data:aerodynamics:aircraft:cruise:oswald_coefficient"]
+        ) ** -1.0
+
+    def compute_partials(self, inputs, partials, discrete_inputs=None):
+        partials[
+            "data:aerodynamics:horizontal_tail:cruise:induced_drag_coefficient",
+            "data:geometry:horizontal_tail:aspect_ratio",
+        ] = -(
+            (
+                np.pi
+                * inputs["data:geometry:horizontal_tail:aspect_ratio"] ** 2.0
+                * inputs["data:aerodynamics:aircraft:cruise:oswald_coefficient"]
+            )
+            ** -1.0
+        )
+
+        partials[
+            "data:aerodynamics:horizontal_tail:cruise:induced_drag_coefficient",
+            "data:aerodynamics:aircraft:cruise:oswald_coefficient",
+        ] = -(
+            (
+                np.pi
+                * inputs["data:geometry:horizontal_tail:aspect_ratio"]
+                * inputs["data:aerodynamics:aircraft:cruise:oswald_coefficient"] ** 2.0
+            )
+            ** -1.0
+        )
+
+
+class WingLowSpeedVectors(om.ExplicitComponent):
+    """
+    Defining the low speed vectors for other computations based on an elliptic distribution
+    assumption.
+    """
+
+    def setup(self):
+        self.add_input("data:geometry:wing:root:chord", val=np.nan, units="m")
+        self.add_input("data:geometry:wing:tip:chord", val=np.nan, units="m")
+        self.add_input("data:geometry:wing:kink:span_ratio", val=np.nan)
+        self.add_input("data:geometry:wing:b_50", val=np.nan, units="m")
+
+        self.add_output(
+            "data:aerodynamics:wing:low_speed:Y_vector",
+            val=np.linspace(0.0, 5.0, SPAN_MESH_POINT),
+            units="m",
+        )
+        self.add_output(
+            "data:aerodynamics:wing:low_speed:chord_vector",
+            val=np.linspace(2.0, 1.0, SPAN_MESH_POINT),
+            units="m",
+        )
+        self.add_output(
+            "data:aerodynamics:wing:low_speed:CL_vector",
+            val=np.full(SPAN_MESH_POINT, 0.5),
+        )
+
+    def setup_partials(self):
+        self.declare_partials(
+            "data:aerodynamics:wing:low_speed:Y_vector",
+            "data:geometry:wing:b_50",
+            val=np.linspace(0, 0.5, SPAN_MESH_POINT),
+        )
+        self.declare_partials(
+            "data:aerodynamics:wing:low_speed:chord_vector",
+            ["data:geometry:wing:tip:chord", "data:geometry:wing:root:chord"],
+            method="exact",
+            rows=np.arange(SPAN_MESH_POINT),
+            cols=np.zeros(SPAN_MESH_POINT),
+        )
+
+    def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
+        b = inputs["data:geometry:wing:b_50"]
+        c_r = inputs["data:geometry:wing:root:chord"]
+        c_t = inputs["data:geometry:wing:tip:chord"]
+        before_kink_point = int(inputs["data:geometry:wing:kink:span_ratio"] * 50)
```

Comment:
Use this instead:
```suggestion
        before_kink_point = inputs["data:geometry:wing:kink:span_ratio"].value * 50
```
Otherwise numpy will be angry we are trying to transform a 1D array into a float ^^'

Reply:
Correction implemented at 8113656e.